lesson08/assignment/video_cli/l8/src/post.vue

```diff
@@ -32,11 +32,11 @@
     props: {
       src: {
         type: String,
-        required: true,
```

Comment:
I just set to these false, but if they are required, you will need to make sure they are set in the data object.